docs/cow-protocol/concepts/flashloans/solvers.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+---
+
+# Solvers
+
+The solver receives for each order an optional object indicating the flashloan's hints. This information serves as a hint for the solver, but the solver ultimately can modify this data in order to make the operation more optimal.
```

Comment:
> the solver ultimately can modify this data in order to make the operation more optimal

Probably better to explain this more concretely. The solver is allowed to return a list of `flashloan` objects in their solution. This allows following options:
* don't provide anything => the driver will assume the flashloan hint attached to orders settled in the solution
* explicitly copy over the flashloan hint provided with the order
* explicitly provide a custom flashloan hint - this is the way how a solver can optimize their use of flashloans